miqa/learning/nn_classifier.py

```diff
@@ -211,7 +211,7 @@ def get_model(file_path=None):
 
     if file_path is not None:
         model.load_state_dict(torch.load(file_path, map_location=device))
-        print(f'Loaded NN model from file "{file_path}"')
+        # print(f'Loaded NN model from file "{file_path}"')
```

Comment:
Is the inclusion of this intentional? Let's either delete the line of code or add it back in rather than leaving it commented. (Same for the other two changes in this file.)